src/firestore/index.ts

```diff
@@ -139,3 +142,58 @@ export function getFirestore(
     'firestore', (app) => new FirestoreService(app));
   return firestoreService.getDatabase(databaseId);
 }
+
+/**
+ * Gets the {@link https://googleapis.dev/nodejs/firestore/latest/Firestore.html | Firestore}
+ * service for the given app, passing extra parameters to its constructor.
+ *
+ * @example
+ * ```javascript
+ * // Get the Firestore service for a specific app, require HTTP/1.1 REST transport
+ * const otherFirestore = initializeFirestore(app, {preferRest: true});
+ * ```
+ *
+ * @param App - whose `Firestore` service to
+ *   return. If not provided, the default `Firestore` service will be returned.
+ * 
+ * @param settings - Settings object to be passed to the constructor.
+ *
+ * @returns The `Firestore` service associated with the provided app.
+ */
+export function initializeFirestore(app: App, settings?: FirestoreSettings): Firestore;
+
+/**
+ * @param app 
+ * @param settings 
+ * @param databaseId 
+ * @internal
+ */
+export function initializeFirestore(
+  app: App,
+  settings: FirestoreSettings,
+  databaseId: string
+): Firestore;
```

Comment:
Did we include `initializeFirestore()` version with `databaseId` in any of the API proposals?

Reply:
No, and I'm not sure if we need it - I just copied the logic from `getFirestore()`. If you think it's not required I'll remove it.

Reply:
I think `databaseId` is an EAP that @tom-andersen worked on. Let's sync up on chat.

Reply:
Just FYI, I'm out till the end of this week, so I'll come back to the chat to discuss this on Monday :) (I'm just replying to comments here for now)